.gitignore

```diff
@@ -1,26 +1,15 @@
-**/target/
-**/.env
-**/.env2
-**/rust-toolchain
-hfuzz_target
-hfuzz_workspace
-**/Cargo.lock
-
-**/*.rs.bk
-
-*.o
-*.so
-*.rlib
-*.dll
-.gdb_history
-
-*.exe
-
+# System
 .DS_Store
 
-.cargo
+# Integrated Development Environment
 .idea
 .vscode
-*.iml
-*.swp
-*.swo
+
+# Package Manager
+## cargo
+/target
+## npm
```

Comment:
This seems useless, all js files are removed.

Reply:
But let's keep this file the same as other repos.